src/Card.js

```diff
@@ -3,11 +3,19 @@ import React from 'react';
 const Card = ( {district} ) => {
   const districtKeys = Object.keys(district.data);
   const yearlyData = districtKeys.map( (districtKey) => {
-    return (
-      <h3 className="data-line">
-        {districtKey}: {district.data[districtKey]}
-      </h3>
-    );
+    if (district.data[districtKey] > .5) {
```

Comment:
we can refactor this into a if/ternary that sets a variable of the className and then use that.